tests/test_utils.py

```diff
@@ -133,6 +134,52 @@ def test_get_editor_precedence():
     assert utils.get_editor(onyo_root) == 'third'
 
 
+def test_generate_faux_serials():
+    """
+    Test the creation of unique serial numbers with the default length through
+    calling the generate_faux_serial() function multiple times and comparing the
+    generated faux serial numbers.
+    """
+    onyo_root = './'
+
+    # technically, creating multiple faux serial numbers without actually
+    # creating assets might lead to duplicates. In practice, this should not
+    # happen if they are sufficiently long and random
+    faux = [utils.generate_faux_serial(onyo_root) for x in range(0, 100)]
+    assert len(faux) == len(set(faux))
+    assert len(faux) > 0
+
+
+def test_length_range_of_generate_faux_serials():
+    """
+    Create many faux serial numbers with different lengths over the range of
+    allowed faux serial number lengths to test that the function returns faux
+    serial numbers, as well as for the values 0 and 38 outside of the interval
+    to verify that the appropriate error gets thrown. Because of the
+    different lengths, they should all be different.
+    """
+    onyo_root = './'
+
+    # test that faux serial numbers must have at least length 1 and the function
+    # raises the correct ValueError:
+    with pytest.raises(ValueError) as ret:
+        utils.generate_faux_serial(onyo_root, faux_length=0)
+    assert ret.type == ValueError
+
+    # test that faux serial numbers can't be requested longer than 37 and the
+    # function raises the correct ValueError:
+    with pytest.raises(ValueError) as ret:
+        utils.generate_faux_serial(onyo_root, faux_length=38)
+    assert ret.type == ValueError
+
+    # technically, creating multiple faux serial numbers without actually
+    # creating assets might lead to duplicates. In practice, this should not
```

Comment:
This comment is unnecessary. They cannot possibly conflict, as they are all different lengths.